frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/StatusField.tsx

```diff
@@ -122,22 +129,45 @@ export function StatusField(props: { cluster: Cluster }) {
             )
             break
         case ClusterStatus.creating:
+            if ( isHybrid ) {
+                hasAction = true
+                Action = () => (
+                    <LogsDownloadButton
+                    id="cluster-logs-button"
+                    agentClusterInstall={agentClusterInstall}
+                    variant={ButtonVariant.link}
+                    />
+                )
+            }            
+            break
```

Comment:
Should be removed as creating status should be treated the same as the 2 below.

Reply:
Changes made